src/main/java/com/minecolonies/coremod/colony/buildings/BuildingHome.java

```diff
@@ -116,6 +116,12 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
         }
     }
 
+    public boolean needsAnything()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1161)